app/src/main/java/com/github/polypoly/app/ui/menu/lobby/GameLobbyActivity.kt

```diff
@@ -210,6 +210,15 @@ class GameLobbyActivity : ComponentActivity() {
         }
     }
 
+    private fun launchGameActivity(packageContext: Context) {
+        val completedLobby = gameLobbyWaitingModel.getGameLobby().value
+        if (completedLobby != null) {
+            GameRepository.game = Game.launchFromPendingGame(completedLobby)
+            val gameIntent = Intent(packageContext, GameActivity::class.java)
+            startActivity(gameIntent)
```

Comment:
You probably should call finish() to ensure nobody can return to a lobby once a game is started